crates/ruff_python_trivia/src/tokenizer.rs

```diff
@@ -119,7 +119,9 @@ pub fn lines_after_ignoring_end_of_line_trivia(offset: TextSize, code: &str) ->
     SimpleTokenizer::starts_at(offset, code)
         .skip_while(|token| token.kind != SimpleTokenKind::Newline && token.kind.is_trivia())
         .take_while(|token| {
-            token.kind == SimpleTokenKind::Newline || token.kind == SimpleTokenKind::Whitespace
+            token.kind == SimpleTokenKind::Newline
```

Comment:
Did you verify that all call sites require the "skipping over semicolons" behavior? I recommend adding tests that exercise each possible call site.